terramate.tm.hcl

```diff
@@ -5,7 +5,7 @@ terramate {
 
     # Optionally disable safe guards
     # Learn more: https://terramate.io/docs/cli/orchestration/safeguards
-    # disable_safeguards = ["git-untracked", "git-uncommitted", "outdated-code"]
+    # disable_safeguards = ["git-untracked", "git-uncommitted", "git-out-of-sync", "outdated-code"]
```

Comment:
@AnnuCode could you put those on multipel lines?

Reply:
like you did here https://github.com/terramate-io/terramate-examples/pull/33/files#diff-46a66afe0363dfbee9d115b47f2317f6953e544033b57e863134e782eef3f084R5